src/logger.py

```diff
@@ -0,0 +1,66 @@
+import logging, sys, os
```

Comment:
pretty sure you didn't mean to include this file in this PR

Reply:
yeah, this is not part of it. When I try to create the other PR it updated this one